packages/mongodb-rag-ingest/src/commands/pages.ts

```diff
@@ -7,23 +7,50 @@ const commandModule: CommandModule<
   Record<string, unknown>,
   LoadConfigArgs & PagesCommandArgs
```

Comment:
for `PageCommandArgs`, like for embedding, should be at sub-command level, not the root